atomix/cluster/src/main/java/io/atomix/raft/partition/RaftPartition.java

```diff
@@ -47,7 +46,6 @@
   private final PartitionId partitionId;
   private final RaftPartitionGroupConfig config;
   private final File dataDirectory;
-  private final ThreadContextFactory threadContextFactory;
```

Comment:
Why we had this factory here before do you know?

Reply:
It was not used in the current code. Not sure what was its use before our refactoring. It was probably something related to client connections.